browser/css/jsdialogs.css

```diff
@@ -369,11 +369,17 @@ algned to the bottom */
 
 .ui-overflow-manager > .notebookbar:not(.ui-separator):not(.ui-overflow-group):not(:has(.ui-overflow-group-container-with-label)) {
 	height: 64px;
-	/* 12px extra margin for containers without label */
-	margin-bottom: 12px;
+	/* 14px extra margin for containers without label */
+	margin-bottom: 14px;
 }
+
+.ui-overflow-manager > .notebookbar.vertical:not(.ui-separator) {
+	height: 64px !important;
```

Comment:
I think we can create variable for that 64px which is used in many places for the same thing: notebookbar element height